ui/src/app/views/concept-set-list/component.html

```diff
@@ -37,16 +37,11 @@
     <div style="display: flex; flex-wrap: wrap;">
       <div *ngFor="let conceptSet of resourceList">
         <app-resource-card [resourceCard]="conceptSet" [cssClass]="'for-list'"
-                           (onUpdate)="loadConceptSets()"
-                           (duplicateNameError)="duplicateNameError($event)">
+                           (onUpdate)="loadConceptSets()">
```

Comment:
should `onUpdate` be in square braces instead of parens?  also do you need to bind loadConceptSets()? (`loadConceptSets.bind(this)`)

Reply:
Note, I didn't add this line, it's just preserving what's there. Since the resource card refactor hasn't happened yet, these are still angular-style bindings.

Reply:
oh riiiight sorry i am working on the resource card refactor now and am used to seeing it that way.